test-case-reporting/index.ts

```diff
@@ -16,11 +16,18 @@
 
 require('dotenv').config();
 
-import {KarmaReporter} from 'test-case-reporting';
+import {Database, dbConnect} from './src/db';
+import {TestResultRecord} from './src/test_result_record';
+import {Travis} from 'test-case-reporting';
 import express from 'express';
+import statusCodes from 'http-status-codes';
+
 const app = express();
 const PORT = process.env.PORT || 8080;
 
+const db: Database = dbConnect();
```

Comment:
nit: you might not have to import `Database` because `const db = dbConnect()` should already provide the type.

Reply:
Done.